### ISSUE TITLE
YAML output needed or confusing?

### ISSUE THREAD

**Issue by mbaudis** [ga4gh-beacon/beacon-v2-unity-testing#2]
The current method converts the source data into YAML and JSON versions, with the JSON version in `json` being thought to be the one for referencing etc..

Since we have now the source in `src` in YAML format[^1] this might be a bit confusing to have a second (converted) YAML in `yaml`. Can we drop this and stay with `src` and `json`? Other structure ideas?

[^1]: As of early March 2022 the YAML in source is still being generated/overwritten by imports from the separate models and framework projects but this will change soon.

**Comment by Tom-Shorter** [comment]
As far as I can see there is no meaningful difference between the contents of the 2 directories (quick diff showed differences only in the descriptions) so keeping both seems to be redundant and getting rid of `yaml` seems sensible.

Can you envision a situation where there will be differences between the directories?

**Comment by mbaudis** [comment]
Not really; especially if the conversion is by action. Just followed the much less complex example of VRS, where there is `vrs-source.yaml`, `vrs.yaml` and `vrs.json`: https://github.com/ga4gh/vrs/tree/main/schema

I'll gladly remove it ...

**Comment by jrambla** [comment]
+1 to remove it

**Comment by mbaudis** [comment]
@jrambla Tada - done.
